lib/integrations/getIntegrations.ts

```diff
@@ -78,6 +79,14 @@ export const ALL_INTEGRATIONS = [
     description: "For personal and business calendars",
     variant: "calendar",
   },
+  {
+    installed: true,
```

Comment:
Do we need some env variables to consider this integration installed? Or does it work out of the box for self-hosted without any previous configuration?

Reply:
yes, will work as long as the exchange server is running, so could be removed

Reply:
@cracker0dks